packages/api/cms-api/src/cron-jobs/jobs.resolver.ts

```diff
@@ -0,0 +1,33 @@
+import { Args, Query, Resolver } from "@nestjs/graphql";
+
+import { CurrentUserInterface } from "../auth/current-user/current-user";
+import { GetCurrentUser } from "../auth/decorators/get-current-user.decorator";
+import { ContentScopeService } from "../content-scope/content-scope.service";
+import { KubernetesService } from "../kubernetes/kubernetes.service";
+import { Job } from "./dto/job.object";
+import { JobsService } from "./jobs.service";
+
+@Resolver(() => Job)
+export class JobsResolver {
+    constructor(
+        private readonly kubernetesService: KubernetesService,
+        private readonly contentScopeService: ContentScopeService,
+        private readonly jobsService: JobsService,
+    ) {}
+
+    @Query(() => [Job])
+    async kubernetesJobs(@Args("cronJob") cronJobName: string, @GetCurrentUser() user: CurrentUserInterface): Promise<Job[]> {
```

Comment:
For me it was not clear from the GraphQL schema that the argument is the name, and not the ID of the cron job. Should we use `cronJobName` instead? What was your intention behind removing the name suffix?

Reply:
You are right, cronJobName is better suited. Fixed in 98cbb58b5af4b089107670562191e31c83a39b93